dubbo-cluster/src/main/java/org/apache/dubbo/rpc/cluster/merger/MapMerger.java

```diff
@@ -30,12 +33,8 @@
         if (ArrayUtils.isEmpty(items)) {
             return Collections.emptyMap();
         }
-        Map<Object, Object> result = new HashMap<Object, Object>();
-        for (Map<?, ?> item : items) {
-            if (item != null) {
-                result.putAll(item);
-            }
-        }
+        Map<Object, Object> result = Arrays.stream(items).filter(Objects::nonNull).map(Map::entrySet)
+                .flatMap(Set::stream).collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue));
```

Comment:
Can these items have same elements? If so, we should provide a merge function here, or it will throws duplicated keys exception when encounter key-value mapping with same key.